integration/test.ats

```diff
@@ -145,23 +147,29 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001,
     if ".json" in str(tolerance):
         tolerance_arg = "--tolerance-file={0}".format(tolerance)
 
-    if should_baseline:
-        # Copy pre-existing normal serac summary file to baseline location
-        copy_args = "{0} {1}".format(normal_test_file, normal_baseline_file)
-        test(
-            executable=serac_copy,
-            clas=copy_args,
-            label=name,
-            nosrun=True)
+    # If should baseline...
+    if serac_baseline != "none":
+        if serac_baseline == "all" or name in serac_baseline:
+            # Remove test from list
```

Comment:
Can you clarify why this is removing it from the list?

Reply:
Because at the end, I'm printing the remaining baselines in the list, in case users type a test wrong they can see it didn't work.